README.md

```diff
@@ -2,21 +2,30 @@
 
 [![Greenkeeper badge](https://badges.greenkeeper.io/probot/autolabeler.svg)](https://greenkeeper.io/)
 
-> a GitHub App built with [probot](https://github.com/probot/probot) that adds labels to Pull Requests based on matched file patterns.
+> A GitHub App built with [probot](https://github.com/probot/probot) that adds labels to Pull Requests based on matched file patterns.
 
 ## Using
 
-Configure by creating a `.github/autolabeler.yml` file with a yaml file in the format of `label: file/path`. 
+Configure by creating a `.github/autolabeler.yml` file with a [yaml file](https://en.wikipedia.org/wiki/YAML) in the format of `label: file/path`. Then add the [Probot Auto Labeler Bot](https://github.com/apps/probot-autolabeler) to your repository.
 
 For example,
+
 ```yaml
 frontend: ["*.js", "*.css", "*.html"]
 backend: ["/app", "*.rb"]
 legal: ["LICENSE*", "NOTICES*"]
 config: .github
 ```
+
 Then if a pull request is opened that has `scripts/widget.js` modified, then the frontend label will be added.
 
+You can see what it looks like when working below;
+![Probot Autolabeler Bot adding labels to a pull request](docs/working.png)
+
+##### Issues with Bot
+
+If you are having issues with the GitHub app not working, please contact [@mithro](mailto:me@mith.ro?subject=Issue%20with%20Probot%20Auto%20Labeler) who is running that.
```

Comment:
I appreciate your enthusiasm, but I don't think we can point to as being the go-to for support for the app 😉 Please remove this line 😄

Reply:
The app at https://github.com/apps/probot-autolabeler is running under my account, so it seems reasonable to point people who are having issues to me rather than bothering the owners of this account.

If you wish to deploy and maintain the bot, I'm more than happy to have you do that!

Reply:
Oh I'm sorry, didn't know that! However, I think it makes more sense to just point folks to creating a new issue. Open source is great for that 👍

Reply:
Agreed, public issue discussions would be nicer.

Reply:
👍 directing them to open an issue would be great!

Reply:
I've made this update.